db/rds.go

```diff
@@ -84,29 +94,39 @@ func SelectUserTaskByID(uid string, task_id string) Task {
 func SelectDiscordID(uid string) string {
 	query := fmt.Sprintf("SELECT discord_id FROM users WHERE uid='%s'", uid)
 	var discord_id string
-	err := db.QueryRow(query).Scan(&discord_id)
+	err := DB.QueryRow(query).Scan(&discord_id)
 	if err != nil {
-		panic(err)
+		return ""
```

Comment:
What is this for? vs. "panic(err)" or just "return"

Reply:
return "" to match type and return instead of panic to make it so a decent error message is returned in the api rather than just an error in logs